# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1525107816 14400
#      Mon Apr 30 13:03:36 2018 -0400
# Branch stable
# Node ID cf84db75ab57d4fc5d18682c6ecf9d2e1f66ea94
# Parent  c8fc547ab5dd07129c444924d869924afeb41f29
Version 4.4.0 released.

* configure.ac (AC_INIT): Set version to 4.4.0.
(OCTAVE_MINOR_VERSION): Now 4.
(OCTAVE_PATCH_VERSION): Now 0.
(OCTAVE_RELEASE_DATE): Set to 2018-04-30.
(NEWS): Set release date.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -1,9 +1,9 @@
-Summary of important user-visible changes for version 4.4 (yyyy-mm-dd):
+Summary of important user-visible changes for version 4.4 (2018-04-30):
 ----------------------------------------------------------------------
 
  ** A graphical Variable Editor has been added to the GUI interface.
     It uses a spreadsheet-like interface for quick, intuitive editing
     of variables.  The Variable Editor is launched by double-clicking
     on a variable name in the Workspace Window or by typing
     "openvar VARIABLE_NAME" in the Command Window.
 
diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -15,34 +15,34 @@ dnl Copyright (C) 1993-2018 John W. Eato
 ### GNU General Public License for more details.
 ###
 ### You should have received a copy of the GNU General Public License
 ### along with Octave; see the file COPYING.  If not, see
 ### <https://www.gnu.org/licenses/>.
 
 ### Initialize Autoconf
 AC_PREREQ([2.65])
-AC_INIT([GNU Octave], [4.3.92], [https://octave.org/bugs.html], [octave],
+AC_INIT([GNU Octave], [4.4.0], [https://octave.org/bugs.html], [octave],
         [https://www.gnu.org/software/octave/])
 
 ### Declare version numbers
 
 dnl Note that the version number is duplicated here and in AC_INIT because
 dnl AC_INIT requires it to be static, not computed from shell variables.
 
 OCTAVE_MAJOR_VERSION=4
-OCTAVE_MINOR_VERSION=3
-OCTAVE_PATCH_VERSION=92
+OCTAVE_MINOR_VERSION=4
+OCTAVE_PATCH_VERSION=0
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION argument.
 OCTAVE_VERSION="$PACKAGE_VERSION"
 
 OCTAVE_COPYRIGHT="Copyright (C) 2018 John W. Eaton and others."
 
-OCTAVE_RELEASE_DATE="2018-04-27"
+OCTAVE_RELEASE_DATE="2018-04-30"
 
 ## The "API version" is used as a way of checking that interfaces in the
 ## liboctave and libinterp libraries haven't changed in a backwardly
 ## incompatible way when loading .oct files.  A better way to do this is with
 ## library versioning, but not all systems support it.
 ##
 ## NOTE: This macro will be removed in a future version of Octave!
 ## If you insist on checking for features using a version number, use the
